app/src/main/java/com/cornellappdev/android/eatery/model/EateryBaseModel.java

```diff
@@ -144,6 +155,32 @@ public void parseEatery(Context context, boolean hardcoded, AllEateriesQuery.Eat
     mArea = CampusArea.fromShortDescription(eatery.campusArea().descriptionShort());
   }
 
+  public void parseCtEatery(Context context, AllCtEateriesQuery.CollegetownEatery ctEatery) {
```

Comment:
Should we merge this method with the existing parseEatery method in EateryBaseModel? Like the same way DiningHall and Cafe Models are done with just leaving certain fields null if they don't apply (in this case CampusArea)

Reply:
the thing with this is that for parseEatery it takes, AllEateriesQuery.Eatery, and it already distinguishes btw hardcoded and non hardcoded in parse eatery so i didnt want to add more logic into one method. i think having a separate method for ct is more clear and cleaner.